### DISCUSSION TITLE
3-8 method?

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#955] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Should I take my Chromebook apart for that 3-8 method or the jumper method cause I have dedede

**Comment by UnicycleParrot** [comment]
what is this 3-8 method you are talking about?
I believe I have dedede too

**Reply by v4pl** [reply, marked as answer]
it involves taking your computer apart and using a paper clip to jump it then you will be able to use s1mmer
